docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md

```diff
@@ -0,0 +1,111 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram/
+description: Configure the Telegram integration to connect alerts generated by Grafana Alerting
+keywords:
+  - grafana
+  - alerting
+  - telegram
+  - integration
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Telegram
+title: Configure Telegram for Alerting
+weight: 300
+---
+
+## Configure Telegram for Alerting
+
+Use the Grafana Alerting - Telegram integration to send [Telegram](https://telegram.org/) notifications when your alerts are firing.
+
+## Before you begin
+
+### Telegram bot API token and chat ID
+
+To integrate Grafana with Telegram, you need to obtain a Telegram **bot API token** and a **chat ID** (i.e., the ID of the Telegram chat where you want to receive the alert notifications).
```

Comment:
where?

Reply:
i can add in-doc links so users don't get lost.

Reply:
you can ignore this. i see you describe it below:)